src/Arcus.WebApi.Unit/Hosting/TestApiServer.cs

```diff
@@ -98,10 +111,13 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
         protected override IWebHostBuilder CreateWebHostBuilder()
         {
             return new WebHostBuilder()
-                .ConfigureAppConfiguration(builder => builder.AddInMemoryCollection(_configurationCollection))
-                .UseStartup<TestStartup>();
+                .ConfigureAppConfiguration(builder => builder.AddInMemoryCollection(_configurationCollection));
         }
 
+#if NETCOREAPP3_0
```

Comment:
Intentional?

Reply:
Woops, no. Thanks. Was a leftover of an overload I tried that was only available in .NET Core 3.0.